crypto-lib/private/libcrypto/ffi.rkt

```diff
@@ -108,11 +108,14 @@
 
 (define (i2d i2d_Type x)
   (define outlen (i2d_Type x #f))
-  (define outbuf (make-bytes outlen 0))
+  ; buffer must not move since pointer passed to i2d_Type does not trace it
+  (define outbuf (malloc outlen 'atomic-interior))
```

Comment:
Well, this isn't `'raw', but it still can't move. It also won't be collected until after the `memcpy` below.